### PR TITLE
[ci] Move proxy login test to rspec test suite

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/spec/features/webui/login_spec.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+require "browser_helper"
+
+RSpec.feature "Login", :type => :feature, :js => true do
+  let!(:user) { create(:confirmed_user, login: "proxy_user") }
```
Comment (ChrisBr) [inline src/api/spec/features/webui/login_spec.rb]: Do we need the exclamation mark?

Comment (bgeuken) [inline src/api/spec/features/webui/login_spec.rb]: We do. Otherwise one of the tests fail.

**Comment by hennevogel** [comment]
Why is this a feature test?

**Comment by bgeuken** [comment]
To verify that users are visibly logged in, which kinda make sense given being able to lock in is quite important. But I am not that opinionated. I can move it to a controller test if you want.

**Comment by hennevogel** [comment]
That a user is able to log in under the different circumstances sounds like a controller test to me...

**Comment by hennevogel** [comment]
Or rather a request spec I guess...

**Comment by hennevogel** [comment]
Oh well anyway, let's do this...

**Pull request merged**